src/mixins/canvas_events.mixin.js

```diff
@@ -812,14 +814,6 @@
         target = this.findTarget(e) || null;
         this._setCursorFromEvent(e, target);
         this._fireOverOutEvents(target, e);
-        // handle triggering on SubTargets
-        this.targets.map(function(subTarget,k){
-          _this._fireOverOutEvents(subTarget, e, '_hoveredTarget' + k);
-        });
-        // hoverCursor should come from top-most subtarget
-        this.targets.slice(0).reverse().map(function(subTarget){
-          _this._setCursorFromEvent(e, subTarget);
-        });
```

Comment:
what do you think about this call to ._setCursorFromEvent call? Can I leave it in, or should it be a separate PR? I feel like it's relevant to add it in here.